src/ConnectedMode/Resources.resx

```diff
@@ -263,13 +263,4 @@
   <data name="ImportBeforeFileGenerator_WritingTargetFileToDisk" xml:space="preserve">
     <value>[ConnectedMode/ImportsBeforeFileGenerator] Writing file to imports before directory</value>
   </data>
```

Comment:
Originally when I added the resource strings we didnt have the MigrationStrings.resx so I moved it in there.